src/Select.js

```diff
@@ -16,7 +16,8 @@ import SelectOption from './SelectOption';
 import OutsideClick from 'ship-components-outsideclick';
 import HighlightClick from 'ship-components-highlight-click';
 
-import dispatch from './dispatch';
+import dispatch from './lib/dispatch';
```

Comment:
Hey, did you test this in any of the apps? I'm not sure the regex we use in webpack will babelify files in sub folders of the apps.

Reply:
I did not. I just tested the examples page which seems to be working.

Reply:
Verified to be working in OneSony against this webpack loader regex: `/ship-components-.*\/src/`

Should be ok to merge up now.